packages/app/src/debug/DebugPageHeader.vue

```diff
@@ -0,0 +1,196 @@
+        >
+          <CommitIcon
+            class="fill-white h-16px w-16px mr-11px"
+          />
+          <span class="sr-only">Commit Hash:</span> {{ defaultscommitInfo.commitHash }}
+        </li>
+        <li
+          v-if="debug?.commitInfo?.authorName"
+          data-cy="debug-header-author"
+        >
+          <i-cy-general-user_x16
+            class="mr-1 icon-dark-gray-500 icon-light-gray-100 icon-secondary-light-gray-200 mr-11px"
+            data-cy="debug-header-avatar"
+          />
+          <span class="sr-only">Commit Author:</span> {{ debug.commitInfo.authorName }}
+        </li>
+        <li
+          v-if="debug.createdAt"
+          data-cy="debug-header-createdAt"
+        >
+          <IconTimeStopwatch
+            size="16"
+            class="mr-9px"
+            stroke-color="gray-500"
+            fill-color="gray-50"
+          />
+          <span class="sr-only">Run Total Duration:</span> {{ debug.totalDuration }} ({{ relativeCreatedAt }})
+        </li>
+      </ul>
+    </div>
+  </div>
+</template>
+<script lang="ts" setup>
+import { computed } from 'vue'
+import DebugResults from './DebugResults.vue'
+import ExternalLink from '@cy/gql-components/ExternalLink.vue'
+import type { DebugPageFragment } from '../generated/graphql'
+import CommitIcon from '~icons/cy/commit_x14'
+import { IconTimeStopwatch } from '@cypress-design/vue-icon'
+import { gql } from '@urql/core'
+import { dayjs } from '../runs/utils/day.js'
+
+interface DefaultcommitInfoTypes {
+  authorName: string
+  branch: string
+  summary: string
+  runNumber: string
+  commitHash: string
+}
+
+interface DefaultTypes {
+  id: string
+  createdAt: string
+  totalDuration: string
+  url: string
+  authorEmail: string
+  commitInfo: DefaultcommitInfoTypes
+}
+
+// the query below will also be updated to fetch debug page information
+// runNumber and commitHash dont exist in the query and should be queried
+gql`
+fragment DebugPage on CloudRun {
+	id
+	createdAt
+	status
+  totalDuration
+  url
+  tags {
+    id
+    name
+  }
+	...RunResults
+	commitInfo {
+		authorName
+		authorEmail
+		summary
+		branch
+	}
+}
+`
+
+const props = defineProps<{
+  gql: DebugPageFragment
+}>()
+
+// These default values exist in case no values are returned by the query
+const defaultscommitInfo: DefaultcommitInfoTypes = {
+  authorName: 'ankithmehta',
+  branch: 'feature/DESIGN-183',
+  summary: 'Adding a hover state to the button component',
+  runNumber: 'Run #468',
+  commitHash: 'b5e6fde',
+}
+
+const defaultValues: DefaultTypes = {
+  id: '3',
+  createdAt: '2022-11-14T14:41:23.078Z',
+  totalDuration: '10m 30s',
+  url: 'cloud.cypress.io',
+  authorEmail: 'hello@cypress.io',
+  commitInfo: defaultscommitInfo,
+}
+
+const debug = computed(() => {
+  if (props.gql) {
+    return props.gql
+  }
+
+  return defaultValues
+})
+
+const relativeCreatedAt = computed(() => dayjs(new Date(debug.value.createdAt!)).fromNow())
+
```

Comment:
Figuring out how to calculate the N commits ahead is going to be fun one (no need to do it here).